programmers/모의고사.js

```diff
@@ -0,0 +1,46 @@
+function solution(answers) {
+    const answerArr = []
+    const supo1 = [1,2,3,4,5];
+    const supo2 = [2,1,2,3,2,4,2,5];
+    const supo3 = [3,3,1,1,2,2,4,4,5,5];
```

Comment:
1, 2, 3 혹은 one, two... 등에 대한 변수는 최대한 지양해야합니다.
이러한 비슷한 형태의 값들을 사용할 때는 하나의 배열로 선언해주세요.

실제 접근할 때는 patterns[0] 으로 접근해서 값을 비교해주시면 됩니다.

추가로, 최근에는 코딩테스트에서도 변수를 어떻게 선언하며 개발하는지 보는 회사들이 늘고 있습니다.
알아두시고, a, b, c와 같은 무자비한 변수명은 피하는 것이 좋아요 ^^
```suggestion
    const patterns = [
        [1,2,3,4,5],
        [2,1,2,3,2,4,2,5],
        [3,3,1,1,2,2,4,4,5,5]
     ];
```